allennlp/data/dataset_reader.py

```diff
@@ -16,6 +19,5 @@ def read(self, file_path: str) -> Dataset:
 
     @classmethod
     def from_params(cls, params: Params):
```

Comment:
Can you add a return type here, and anywhere else on these where it's missing one?